mods/default/functions.lua

```diff
@@ -346,3 +346,40 @@ minetest.register_abm({
 	end
 })
 
+--
+-- Water mist 
+--
+
+minetest.register_abm({
+	nodenames = {"default:water_flowing"},
+	neighbors = {"default:water_source"},
+	interval = 5,
+	chance = 1,
+	action = function(pos, node, active_object_count, active_object_count_wider)
+		local posa = {x=pos.x, y=pos.y+1, z=pos.z}
+		local posu = {x=pos.x, y=pos.y-1, z=pos.z}
+		local na =  minetest.get_node(posa).name
+		local nu =  minetest.get_node(posu).name
+		if na == "default:water_flowing" then
+			if nu == "default:water_source" then
```

Comment:
!= is doesn't equal in c/c++ i think SN  meant ~= but same thing just
different lang
On Oct 16, 2015 11:12 AM, "HybridDog" <notifications@github.com> wrote:

> In mods/default/functions.lua
> <https://github.com/minetest/minetest_game/pull/672#discussion_r42253544>:
>
> > +--
> > +-- Water mist
> > +--
> > +
> > +minetest.register_abm({
> > +	nodenames = {"default:water_flowing"},
> > +	neighbors = {"default:water_source"},
> > +	interval = 5,
> > +	chance = 1,
> > +	action = function(pos, node, active_object_count, active_object_count_wider)
> > +		local posa = {x=pos.x, y=pos.y+1, z=pos.z}
> > +		local posu = {x=pos.x, y=pos.y-1, z=pos.z}
> > +		local na =  minetest.get_node(posa).name
> > +		local nu =  minetest.get_node(posu).name
> > +		if na == "default:water_flowing" then
> > +			if nu == "default:water_source" then
>
> What does != do?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/minetest/minetest_game/pull/672/files#r42253544>.
>